back-end/src/sciler/handler/handler_helper.go

```diff
@@ -76,6 +77,25 @@ func (handler *Handler) SendComponentInstruction(clientID string, instructions [
 	}
 }
 
+// PrepareMessage scans a message and if the instruction is of type status, if so the value of the message is replaced by the status of a device
+func (handler *Handler) PrepareMessage(typeID string, messages []config.ComponentInstruction) []config.ComponentInstruction {
```

Comment:
are the deviceID and componentID already checked at this stage?

Reply:
yes